README.md

```diff
@@ -158,4 +158,22 @@ Once the Pull Request is created, manual action should be perform to:
 - If no changes are detected, the workflow will silently skip the bump.
 
 
+## Debugging job (Post-Sonatype Migration)
+
+This workflow is designed for testing the Maven Central publishing pipeline after the migration to Sonatype. 
```

Comment:
"This workflow is designed for testing the Maven Central publishing pipeline after the migration to Sonatype. " ->
"This workflow is designed to release a snapshot on Maven Central and to validate under the hood that signing an artifact is working".